quickstarts/golden-signals-dashboard/config.yml

```
@@ -0,0 +1,65 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: golden-signals-dashboard-for-New-Relic
```

The slug needs to be lower case:
```suggestion
slug: golden-signals-dashboard-for-new-relic
```